tools/tokenserver/database.py

```diff
@@ -101,6 +101,26 @@
     :offset
 """)
 
+_GET_OLD_USER_RECORDS_FOR_SERVICE_RANGE = """\
+select
+    uid, email, generation, keys_changed_at, client_state,
+    nodes.node, nodes.downed, created_at, replaced_at
+from
+    users left outer join nodes on users.nodeid = nodes.id
+where
+    users.service = :service
+and
+    ::RANGE::
```

Comment:
nit: I'd prefer adding this to `_GET_OLD_USER_RECORDS_FOR_SERVICE` and only adding the entire `and ` clause when needed but not going to block on this

Reply:
I thought about doing that, but since I'm basically doing a string replacement inside of the sql query, (and since this is technically only required for the bulk cleanup we're doing) I figured that it would be cleaner to break these apart.